spec/requests/user_request_spec.rb

```diff
@@ -19,4 +19,22 @@
     expect(response[:data]).to be_a(Hash)
     expect(response[:data][:users][0][:id].to_i).to be_a(Integer)
   end
+
+  it 'handles variables' do
+    post '/graphql', params: { query: '{ users {id, username } }', variables: 1000 }
```

Comment:
this hits the variables:string part of graphql controller

Reply:
Oo interesting, I don't even know how this works!